src/Sylius/Behat/Context/Ui/Admin/ManagingPromotionsContext.php

```diff
@@ -617,6 +620,30 @@ public function thePromotionsShouldHavePriority(PromotionInterface $promotion, i
         Assert::same($this->updatePage->getPriority(), $priority);
     }
 
+    /**
+     * @When /^I want to manage (this promotion)'s coupons$/
+     */
+    public function iWantToManageThisPromotionSCoupons(): void
+    {
+        Assert::true($this->updatePage->isCouponManagementAvailable());
```

Comment:
This is the step when we should call some function on the `updatePage`, like `manageCoupons()`, that in the UI context click on the "Manage coupons" button and therefore redirect us to the coupons management page